pkgs/development/compilers/llvm/11/compiler-rt/default.nix

```diff
@@ -59,8 +60,9 @@ stdenv.mkDerivation {
     # extra `/`.
     ./normalize-var.patch
     ../../common/compiler-rt/libsanitizer-no-cyclades-11.patch
-  ] ++ lib.optional stdenv.hostPlatform.isAarch32 ./armv7l.patch;
-
+    ../../common/compiler-rt/darwin-plistbuddy-workaround.patch
+    ./armv7l.patch
```

Comment:
Why is this patch now here unconditionally?

Reply:
See: https://github.com/NixOS/nixpkgs/pull/186575#pullrequestreview-1074122694